migrations/sql/V2019.01.17.12.10__create_minespace_user_tables.sql

```diff
@@ -1,7 +1,9 @@
-CREATE TABLE minespace_user (
+drop table if exists minespace_user;
+CREATE TABLE minespace_user
+(
     user_id serial PRIMARY KEY,
     keycloak_guid uuid,
-    email character varying(100) NOT NULL, 
+    email character varying(100) NOT NULL,
```

Comment:
Just thought of this now... is 100 characters going to be enough for email addresses? I think the max length allowable is 254 characters... so we may want to consider upping the number on this, eventually.

Reply:
Also, I think it was mentioned in an earlier PR, but should we add the create_user, create_timestamp, update_user, and update_timestamp columns?

Reply:
Hi Kristen! For this PR I'm just renaming this script which was already commited to match our new versioning scheme. Jason is still working on user changes in another branch. I'll pass on your comments to him.

Reply:
Gotcha! Ok thanks. Apologies for commenting in the wrong PR :)

Reply:
No, it's good to raise things when you see them. I did pass your comments to Jason.

Reply:
@klefler 
Didn't know about the 254 email for bceid, we should change it to match the bceid/idir standard. 

These tables are built to support (and eventually be moved to) a separate application, so I didn't automatically apply that same pattern, not sure what we want to do here.